### DISCUSSION TITLE
durexForth 5

### DISCUSSION THREAD

**Discussion by jkotlinski** [jkotlinski/durexforth#537] [General]
It's been a soon a year since last release, so wondering if what is now in master seems good to go for a 5.0.0?

**Comment by Whammo** [comment]
I think the IEC stuff I slipped in there is nice as a jumping off place for experimentation with devices. I'm beginning to doubt there is any other actual use for the words besides prototyping for assembly routines that won't be necessarily faster. The bottleneck is not Kernal files, it's the transfer speed.
 So, however cool I think it might be, I'm not convinced of the usefulness of it to other users.
It would not hurt my feelings if it were determined that it's disk space were best reserved for other uses.

**Reply by jkotlinski** [reply]
I do not have much of an opinion on the IEC code. Personally, I do not see that I have a need for it soon, but maybe someone else will find it interesting and instructive? There is plenty of disk space, that is not to worry about.

**Reply by Whammo** [reply]
All I can say is, durexForth makes it easy. Thank you for durexForth.

**Reply by jkotlinski** [reply]
Thank you for contributing :-)
